core/src/css/test/theme-ionic/typography.e2e.ts

```diff
@@ -0,0 +1,185 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+/**
+ * Text does not have per-mode styles
+ */
+configs({ directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
```

Comment:
We need to change this to the following to run it only for the `ionic` theme:

```suggestion
configs({ modes: ['ionic-md'], directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
```

Reply:
This was recently added and you're one of the first to use it! 🎉